.gitignore

```diff
@@ -0,0 +1 @@
+/venv/idea/sqlite3/pyc/__pycache__/
```

Comment:
Your gitignore format is invalid
Each item should be in a separate line
View example https://djangowaves.com/tips-tricks/gitignore-for-a-django-project/